kafka/source/README.md

```diff
@@ -51,3 +51,68 @@ event sink.
 
 A more detailed example of the `KafkaSource` can be found in the
 [Knative documentation](https://knative.dev/docs/eventing/samples/).
+
+## Experimental KEDA support in Kafka Event Source
+
+Warning: this is *experimental* and may be changed in future. Should not be used in production. This is mainly for discussion and evolving scaling in Knative eventing.
+
+The code is using Unstructured and also imported KEDA API - this is for discussion which version should be used (right now only Unstructured is fully implemented).
+KEDA to provide client-go support discussion #494 <https://github.com/kedacore/keda/issues/494>
+
+### Install Kafka
+
+See above
+
+### Install KEDA -
+
+To install the version I used for the experiment:
+
+```bash
+git clone https://github.com/kedacore/keda.git
+cd keda
+git pull
+git checkout master
+#$ git rev-parse HEAD
+#0099c102d538995c81610fb48d12bde1259678a6
+git checkout 0099c102d538995c81610fb48d12bde1259678a6
+```
+
+### Run Kafka Source Controller
+
+Install Kafka source controller with Keda support:
+
+```bash
+export KO_DOCKER_REPO=...
+ko apply -f kafka/source/config/
+```
+
+#### Local testing
+
+```bash
+go run kafka/source/cmd/controller/main.go -kubeconfig $KUBECONFIG
+```
+
+### Create Kafka Source that uses YAML
+
+And use example YAML that has minReplicaCount set to value different from 1. For example:
+
+```yaml
```

Comment:
Can we provide a full, self-contained example here with the sink that uses a proper `ref` statement?  As well as the matching service definition.  Ideally they'd be able to directly `ko` the sample yaml

Reply:
Provided using ksvc and then added section about testing without using serving